src/main/java/org/candlepin/model/OwnerCurator.java

```diff
@@ -154,11 +154,13 @@ public Owner getByKey(String key) {
     }
 
     @Transactional
-    public CandlepinQuery<Owner> getByKeys(Collection<String> keys) {
-        DetachedCriteria criteria = this.createSecureDetachedCriteria()
-            .add(CPRestrictions.in("key", keys));
+    public List<Owner> getByKeys(Collection<String> keys) {
```

Comment:
Same comment as `DeletedConsumerCurator.findByDate`

Reply:
See the comment for DeletedConsumerResource